journalism_workflow/workflow.py

```diff
@@ -0,0 +1,134 @@
+from orchestra.workflow import Step
+from orchestra.workflow import Workflow
+
+from journalism_workflow.adjust_photos import autoadjust_photos
+from journalism_workflow.documents import create_documents
+
+# Instantiate our workflow
+journalism_workflow = Workflow(
+    slug='journalism',
+    name='Journalism Workflow',
+    description='Create polished newspaper articles from scratch.',
+)
+
+# Zeroth step: we autocreate subfolders and documents we'll need.
+document_creation = Step(
+    slug='document_creation',
+    name='Document Creation',
+    description='Create google documents for project in relevant folder.',
+    worker_type=Step.WorkerType.MACHINE,
+    creation_depends_on=[],
+    function=create_documents,
+)
+journalism_workflow.add_step(document_creation)
+
+# First step: an editor plans out the high-level story idea.
+editor_step = Step(
+    slug='editor',
```

Comment:
i wonder if the slugs should be closer to the names, rather than the certifications?